src/odemis/gui/main_xrc.py

```diff
@@ -5491,16 +5490,15 @@ def __init_resources():
                               <flag>wxTOP|wxBOTTOM|wxRIGHT</flag>
                               <border>14</border>
                             </object>
-
                             <orient>wxHORIZONTAL</orient>
                           </object>
                         </object>
-                        <label>ALIGNMENT</label>
+                        <label>CALIBRATION 1</label>
```

Comment:
I think this should be `<label>CALIBRATION 2</label>`, otherwise it doesn't match line 5510. And on line 5501 you also have `pnl_fastem_calibration_2`

Reply:
This line is still referring to the item in line 5443. Item in line 5510 is referring to line 5501.